### ISSUE TITLE
Reloading a suite with files removed doesn't remove such files from the cylc run directory

### ISSUE THREAD

**Issue by DJDavies2** [metomi/rose#2477]
I start a suite, I am using rose stem here but I think it makes no dfference whether it is rose stem or rose suite-run.  I then remove a file in the local rose directory, in this case a script in an app/task/bin directory.  I then reload the suite via .... --reload.  The file I removed locally is still present in the cylc-run directory.

This is confusing and also potentially breaking as for example I could have logic that relies on testing for the existence of files elsewhere in the suite.  If I  have such logic it will fail when it should succeed and will succeed if I run the suite from scratch.  As such I think this should be seen as a bug.

**Comment by oliver-sanders** [comment]
Unfortunately this is the designed behaviour of `rose suite-run` (it is effectively a `rsync` that doesn't use the `--delete` argument).

It partly comes down to the difficulty of determining whether a file in the run dir was put there by a previous run of `rose suite-run` or by a task run by the workflow.

With the next versions of Rose and Cylc (currently in beta, due soon) we have changed things. The `rose suite-run` command has been replaced by `cylc install`. The new command *does* remove old files (using `rsync --delete`), there is a new configuration file which can be used to configure things that `cylc install` should not install.
